xml/schema/types/oblocks-2-9-6.xsd

```diff
@@ -43,9 +43,9 @@
         <xs:element name="oblock" minOccurs="0" maxOccurs="unbounded">
           <xs:complexType>
             <xs:sequence>
```

Comment:
My understanding is that xs:all allows child elements to have unbounded maxOccurs.